app/src/main/java/fr/neamar/kiss/notification/NotificationListener.java

```diff
@@ -80,45 +87,53 @@ public void onListenerDisconnected() {
         for (String packageName : packages) {
             editor.remove(packageName);
         }
+
         editor.apply();
 
+        Log.v(TAG, "Removed all notifications for " + packages.toString());
+
         super.onListenerDisconnected();
     }
 
+    @Override
+    public void onNotificationRankingUpdate(RankingMap rankingMap) {
```

Comment:
can this impact our dots? It doesn't seem reordering can have an impact

Reply:
This should have no impact, but it is even necessary if notification channel changes.
onNotificationRankingUpdate is callend if any property of a notification channel is changed, also canShowBadge.
But then the notification itself is not posted or removed again, the only thing that changes is this rankingMap from where we can read canShowBadge.
This may affect any shown notification, so i went the easy way here and just handled all notifications again like when onListenerConnected is called.
Hopefully this is ok.